### DISCUSSION TITLE
Ignoring linefeeds?

### DISCUSSION THREAD

**Discussion by musicpanda** [WinMerge/winmerge#1370] [General]
A problem I regularly encounter is that when I compare two files they are marked as completely different by WinMerge.

However, when I have a look the texts look the same. So all the difference must be in different linefeeds and/or different spacing (spaces vs tabs). 

Is there a setting so that those linefeeds and spacing are ignored and I just see the textual difference?

**Comment by sdottaka** [comment]
To ignore differences in whitespace and EOL, please select Whitespaces→Ignore changes or Ignore all menu items and Ignore carriage return differences menu item as shown below.

![image](https://user-images.githubusercontent.com/98126/174073277-f0675d95-a0be-460c-9346-596b13fd91d1.png)
![image](https://user-images.githubusercontent.com/98126/174073106-d30577bf-d0d9-487b-8c8a-ceed77715ef4.png)
